### ISSUE TITLE
Follow up: Download instead of subscription page is shown for dowjones referral promo

### ISSUE THREAD

**Issue by btlechowski** [brave/brave-browser#5663]
This is a regression. Original issue is here: https://github.com/brave/brave-browser/issues/1805

## Steps to Reproduce

1. Clean install. 
2. Download the installer from `https://brave.com/dowjones-barrons`
3. Run the installer
4. After Brave has started wait for `https://brave.com/dowjones-barrons` page to be shown (can take up to 60 seconds)

## Actual result:

The Download page is shown.

![image](https://user-images.githubusercontent.com/34715963/63041269-bf5fb280-bec7-11e9-95f7-a273ba274ffc.png)

## Expected result:
The Subscription page is shown
![image](https://user-images.githubusercontent.com/34715963/47383295-b4ebe680-d704-11e8-81a2-b6a1522e9914.png)

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

Brave | 0.67.125 Chromium: 76.0.3809.100 (Official Build) (64-bit)
-- | --
Revision | ed9d447d30203dc5069e540f05079e493fc1c132-refs/branch-heads/3809@{#990}
OS | Windows 7 Service Pack 1 (Build 7601.24494)

cc @brave/legacy_qa @rebron

**Comment by AlanBreck** [comment]
Is this the same bug? https://github.com/brave/brave-com-resources/issues/72

**Comment by btlechowski** [comment]
@AlanBreck It is the same issue as in brave/brave-com-resources#72

**Comment by AlanBreck** [comment]
I'm unable to replicate in that https://brave.com/dowjones-barrons never shows after opening the newly installed Brave. Is this because I've already had Brave installed and there's app data hanging around somewhere?

**Comment by rebron** [comment]
@AlanBreck Yes. You'd need to delete your browser profile completely and clean install. https://github.com/brave/brave-browser/issues/1805#issue-373149801
